package.json

```diff
@@ -30,9 +30,9 @@
     "eslint-plugin-react": "^2.3.0",
     "express": "^4.13.3",
     "gh-pages": "^0.4.0",
-    "react-transform-catch-errors": "^1.0.0",
-    "react-transform-hmr": "^1.0.0",
-    "redbox-react": "^1.0.1",
+    "react-transform-catch-errors": "1.0.0",
```

Comment:
Newer versions of these packages seem to break something. Removing the `^` was a fix found [here](http://stackoverflow.com/a/37896786)